<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1042</width>
    <height>560</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Terminal for Tentacle Mini pH meter</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>550</width>
      <height>471</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideNone</enum>
    </property>
    <property name="usesScrollButtons">
     <bool>false</bool>
    </property>
    <property name="documentMode">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="mainTab">
     <attribute name="title">
      <string>Main</string>
     </attribute>
     <widget class="QPushButton" name="btnLogStop">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>394</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Stop Logging</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>356</y>
        <width>110</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Insert Comment</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="leComment">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>356</y>
        <width>401</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnLogStart">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>394</y>
        <width>110</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Start logging</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="contCB">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>326</y>
        <width>110</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Continuous</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="EZOTab1">
     <attribute name="icon">
      <iconset resource="../atlasterminal.qrc">
       <normaloff>:/new/images/images/ph-circuit-large.jpg</normaloff>:/new/images/images/ph-circuit-large.jpg</iconset>
     </attribute>
     <attribute name="title">
      <string>pH1</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="EZOTab2">
     <attribute name="icon">
      <iconset resource="../atlasterminal.qrc">
       <normaloff>:/new/images/images/ph-circuit-large.jpg</normaloff>:/new/images/images/ph-circuit-large.jpg</iconset>
     </attribute>
     <attribute name="title">
      <string>pH2</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="logTab">
     <attribute name="icon">
      <iconset resource="../atlasterminal.qrc">
       <normaloff>:/new/images/images/log-file-format.png</normaloff>:/new/images/images/log-file-format.png</iconset>
     </attribute>
     <attribute name="title">
      <string>Logging</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1042</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCalls">
    <property name="title">
     <string>Calls</string>
    </property>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionScreenshot"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionIndex"/>
    <addaction name="separator"/>
    <addaction name="actionAbout_AtlasTerminal"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTools"/>
   <addaction name="menuCalls"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionQuit"/>
   <addaction name="actionConfigure"/>
   <addaction name="actionConnect"/>
   <addaction name="actionDisconnect"/>
   <addaction name="action_Help_Tentacle"/>
  </widget>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="../atlasterminal.qrc">
     <normaloff>:/new/images/images/application-exit.png</normaloff>:/new/images/images/application-exit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="icon">
    <iconset resource="../atlasterminal.qrc">
     <normaloff>:/new/images/images/connect.png</normaloff>:/new/images/images/connect.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Connect</string>
   </property>
   <property name="toolTip">
    <string>Connect Atlas</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="icon">
    <iconset resource="../atlasterminal.qrc">
     <normaloff>:/new/images/images/disconnect.png</normaloff>:/new/images/images/disconnect.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Disconnect</string>
   </property>
   <property name="toolTip">
    <string>Disconnect Atlas</string>
   </property>
  </action>
  <action name="actionConfigure">
   <property name="icon">
    <iconset resource="../atlasterminal.qrc">
     <normaloff>:/new/images/images/settings.png</normaloff>:/new/images/images/settings.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Configure</string>
   </property>
   <property name="toolTip">
    <string>Configure Atlas</string>
   </property>
  </action>
  <action name="action_Help_Tentacle">
   <property name="icon">
    <iconset resource="../atlasterminal.qrc">
     <normaloff>:/new/images/images/AtlasHelp1.png</normaloff>:/new/images/images/AtlasHelp1.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Help Tentacle</string>
   </property>
  </action>
  <action name="actionScreenshot">
   <property name="text">
    <string>Screenshot</string>
   </property>
  </action>
  <action name="actionIndex">
   <property name="text">
    <string>Index</string>
   </property>
  </action>
  <action name="actionAbout_AtlasTerminal">
   <property name="text">
    <string>About AtlasTerminal</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../atlasterminal.qrc"/>
 </resources>
 <connections/>
</ui>
